tests/testthat/test-grandStandardizedSolution.R

```diff
@@ -0,0 +1,193 @@
+library(lavaan)
+# - [ ] add unit tests for std_beta_est()
+
+# Test grandStardardizedSolution -----------------------------------------------
+
+## Maximum expected error of estimates
+err <- .0001
+
+## Single-group, two-factor
+
+mod1 <- '
+   # latent variables
+     ind60 =~ x1 + x2 + x3
+     dem60 =~ y1 + y2 + y3 + y4
+   # regressions
+     dem60 ~ ind60
+'
+fit1 <- sem(model = mod1,
+            data  = PoliticalDemocracy)
+s2_std_beta <- grandStardardizedSolution(fit1)
+
+### lavaan::standardizedSolution()
+s2_std_beta_lav <- subset(standardizedSolution(fit1), op == "~")
+
+test_that("Test for single group warning message",
+{expect_message(grandStardardizedSolution(fit1), "The grand standardized solution is equivalent to the standardizedSolution() from lavaan for a model with a single group.", fixed=TRUE)})
+
+test_that("Standardized beta in a model with single group, two factors",
+          { expect_equal(s2_std_beta$est.std, s2_std_beta_lav$est.std) })
+test_that("SE of standardized beta in a model with single group, two-factors",
+          { expect_equal(s2_std_beta$se,  s2_std_beta_lav$se) })
+
+## Single-group, three-factor
+
+mod2 <- '
+    # latent variables
+      ind60 =~ x1 + x2 + x3
+      dem60 =~ y1 + y2 + y3 + y4
+      dem65 =~ y5 + y6 + y7 + y8
+    # regressions
+      dem60 ~ ind60
+      dem65 ~ ind60 + dem60
+'
+fit2 <- sem(model = mod2,
+            data  = PoliticalDemocracy)
+s3_std_beta <- grandStardardizedSolution(fit2)
+
+### lavaan::standardizedSolution()
+s3_std_beta_lav <- subset(standardizedSolution(fit2), op == "~")
+
+apply(rbind(s3_std_beta_lav$est.std, s3_std_beta$est.std), 2,
```

Comment:
Wouldn't `test_that(code = { expect_equal(s3_std_beta_lav$est.std, s3_std_beta$est.std)})` be simpler? They are using the same functions anyway.

Reply:
Yes it does (for some reason it didn't work before but now it does). Thanks!